graylog2-server/src/main/java/org/graylog2/configuration/WebListenUriValidator.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.configuration;
+
+import com.github.joschi.jadconfig.ValidationException;
+import com.github.joschi.jadconfig.Validator;
+
+import java.net.URI;
+
+public class WebListenUriValidator implements Validator<URI> {
```

Comment:
Just FYI, this could also be done with a [`ValidatorMethod`](https://joschi.github.io/JadConfig/0.12.2/apidocs/com/github/joschi/jadconfig/ValidatorMethod.html) directly in `Configuration `, but having a dedicated validator is also fine.